### DISCUSSION TITLE
release 里面看不到v1.37.1?

### DISCUSSION THREAD

**Discussion by mio-19** [electerm/electerm#3122] [General]
![image](https://github.com/electerm/electerm/assets/57285379/c4b40c0a-d509-4ada-8290-cac8d6cce689)

**Comment by zxdong262** [comment]
可能缓存导致的 可以从主页查看 https://electerm.html5beta.com/

**Comment by zxdong262** [comment]
确实我退出登录就看不到，很奇怪，也许github数据错误

**Comment by zxdong262** [comment]
有人遇到同样问题 https://github.com/orgs/community/discussions/72846
